src/components/common/index.js

```diff
@@ -7,26 +7,29 @@ export const createDisabledTextStyles = () => css`
   color: ${({ theme }) => theme.textDisabled};
   text-shadow: 1px 1px ${({ theme }) => theme.textDisabledShadow};
   /* filter: grayscale(100%); */
+  -webkit-text-fill-color: ${({ theme }) => theme.inputTextDisabled};
```

Comment:
In my opinion this is a better location to fix this issue, instead of writing it twice into separate components. Other components using the `createDisabledTextStyles()` function shouldn't be affected by this change.